packages/react-ui/src/features/flows/components/flow-status-toggle.tsx

```diff
@@ -22,25 +34,58 @@ export default function FlowStatusToggle({ flow }: { flow: Flow }) {
           status: checked ? FlowStatus.ENABLED : FlowStatus.DISABLED,
         },
       });
+      setIsChecked(checked);
     } finally {
       setIsLoading(false);
     }
   };
 
+  const renderIcon = (flow: PopulatedFlow) => {
```

Comment:
This won't be needed If you make the funciton return react component, also please don't use inline colors this won't work in darkmode.

You can use the following classes instead: text-background and fill-background